.github/workflows/ci.yml

```diff
@@ -0,0 +1,322 @@
+name: LinuxKit CI
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build & Test
+    strategy:
+      matrix:
+        arch:
+          - amd64-linux
+          - arm64-linux
+          - s390x-linux
+          - amd64-darwin
+          - amd64-windows.exe
+
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.11
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.11
+      id: go
```

Comment:
Pretty old version, is it not?

Reply:
Yes it is, but it's what's currently being used in CI. I didn't want to bump it here.

Reply:
That is a very good point. I think I will open an issue separately.